packages/xo-web/src/common/xo/index.js

```diff
@@ -359,7 +359,18 @@ export const subscribeRemotes = createSubscription(() => _call('remote.getAll'))
 
 export const subscribeRemotesInfo = createSubscription(() => _call('remote.getAllInfo'))
 
-export const subscribeProxies = createSubscription(() => _call('proxy.getAll'))
+export const subscribeProxies = createSubscription(async () => {
+  const { user } = store.getState()
```

Comment:
If `proxy.getAll` throw, then the subscription will no longer throw. Try to keep the same behavior as before, you don't need to log the error.

```js
import { isAdmin } from 'selectors'
...
const isAdmin = isAdmin(store.getState())
return isAdmin ? _call('proxy.getAll') : undefined